src/app/components/mnoe-organizations-local-list/mnoe-organizations-local-list.coffee

```diff
@@ -68,35 +68,6 @@
       else
         setSearchOrganizationsList()
 
-    scope.editRole = (organization) ->
-      # Keep track of old roles when editing organization's roles.
-      organization.beforeEditRole = organization.role
-      organization.editMode = true
-
-    scope.exitEditRole = (organization) ->
-      organization.role = organization.beforeEditRole
-      organization.editMode = false
-
-    scope.updateUserRole = (organization, user) ->
-      user.isUpdatingRole = true
-      # The role must be set on the user for #updateUserRole.
-      user.role = organization.role
-      MnoeUsers.updateUserRole(organization, user).then(
```

Comment:
Remove unused dependencies from this file - `MnoeUsers, UserRoles, toastr, $translate`
Can also get rid of line 15 in this file => `scope.userRoles = UserRoles` since this is no longer used.